### ISSUE TITLE
Got the error "Uncaught Error: Cannot find module 'js-yaml' "  on windows

### ISSUE THREAD

**Issue by sundapeng** [zhuochun/md-writer#11]
Great package, but it seems the package doesn't work on windows.

**Comment by RafaAguilar** [comment]
I know this was closed, but with 0.176.0 (on Windows) still has the same issue:

```
module.js:346
Hide Stack Trace
Error: Cannot find module 'js-yaml'
  at Module._resolveFilename (module.js:344:15)
  at Function.Module._resolveFilename (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\src\module-cache.js:383:52)
  at Function.Module._load (module.js:286:25)
  at Module.require (module.js:373:17)
  at require (module.js:392:17)
  at Object.<anonymous> (c:\Users\raguilar\.atom\packages\md-writer-master\lib\utils.coffee:3:8)
  at Object.<anonymous> (c:\Users\raguilar\.atom\packages\md-writer-master\lib\utils.coffee:1:1)
  at Module._compile (module.js:468:26)
  at Object.requireCoffeeScript [as .coffee] (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\src\coffee-cache.js:74:19)
  at Module.load (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\node_modules\coffee-script\lib\coffee-script\register.js:45:36)
  at Function.Module._load (module.js:318:12)
  at Module.require (module.js:373:17)
  at require (module.js:392:17)
  at Object.<anonymous> (c:\Users\raguilar\.atom\packages\md-writer-master\lib\style-line.coffee:1:9)
  at Object.<anonymous> (c:\Users\raguilar\.atom\packages\md-writer-master\lib\style-line.coffee:1:1)
  at Module._compile (module.js:468:26)
  at Object.requireCoffeeScript [as .coffee] (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\src\coffee-cache.js:74:19)
  at Module.load (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\node_modules\coffee-script\lib\coffee-script\register.js:45:36)
  at Function.Module._load (module.js:318:12)
  at Module.require (module.js:373:17)
  at require (module.js:392:17)
  at atom-workspace.<anonymous> (c:\Users\raguilar\.atom\packages\md-writer-master\lib\main.coffee:78:26)
  at atom-workspace.handler (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\src\space-pen-extensions.js:115:32)
  at atom-workspace.jQuery.event.dispatch (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\node_modules\space-pen\vendor\jquery.js:4681:9)
  at atom-workspace.elemData.handle (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\node_modules\space-pen\vendor\jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\src\command-registry.js:243:29)
  at c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\src\command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\node_modules\command-palette\lib\command-palette-view.js:120:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\node_modules\archive-view\node_modules\atom-space-pen-views\lib\select-list-view.js:335:21)
  at space-pen-div.atom.commands.add.core:confirm (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\node_modules\archive-view\node_modules\atom-space-pen-views\lib\select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\src\command-registry.js:243:29)
  at c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\src\command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\node_modules\atom-keymap\lib\keymap-manager.js:558:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\node_modules\atom-keymap\lib\keymap-manager.js:396:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (c:\Users\raguilar\AppData\Local\atom\app-0.176.0\resources\app\src\window-event-handler.js:170:20)
```

_note: the installation was manually 'cause i'm behind a firewall_

**Comment by zhuochun** [comment]
Can you re-install the updated version?
